SparkPost/matchbox
# Resolves #301 - Adds Prettier support

Resolves #301 

### What Changed
- Adds Prettier support
- Tests new plugin by making an arbitrary change, then running Prettier both in the precommit hook and on save

### How To Test or Verify
- Checkout the `incorporate-prettier` branch, go to a file, make an arbitrary change
- On commit, the precommit hook should format the file
- If using VSCode, install the `prettier-vscode` extension (https://github.com/prettier/prettier-vscode). On save, Prettier should now format files

### PR Checklist
N/A
